packages/nextjs/components/stake/StakeContractInteraction.tsx

```diff
@@ -49,23 +50,26 @@ export const StakeContractInteraction = ({ address }: { address?: string }) => {
     functionName: "completed",
     watch: true,
   });
+
   const { data: myStake } = useScaffoldReadContract({
     contractName: "Staker",
     functionName: "balances",
     args: [connectedAddress ?? ""],
     watch: true,
   });
 
-  const { writeAsync: execute } = useScaffoldWriteContract({
+  const { sendAsync: execute } = useScaffoldWriteContract({
     contractName: "Staker",
     functionName: "execute",
   });
-  const { writeAsync: withdrawETH } = useScaffoldWriteContract({
+
+  const { sendAsync: withdrawETH } = useScaffoldWriteContract({
     contractName: "Staker",
     functionName: "withdraw",
+    args: []
```

Comment:
The new changes in useScaffoldWriteContract give me error if I don't add the `args: []`

![Captura desde 2024-10-15 22-40-28](https://github.com/user-attachments/assets/49696339-2252-4bab-a0e7-2a7ea95edae7)
![Captura desde 2024-10-15 22-39-49](https://github.com/user-attachments/assets/92c75d93-2dca-4f1e-9840-abdef713d8b7)